content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md

```diff
@@ -798,53 +798,100 @@ updates:
+            <td rowspan="3" align="center"><code>1.0.0</code></td>
+            <td rowspan="3" align="center"><code>2.0.0</code></td>
+            <td align="center"><code>'widen'</code></td>
+            <td><code>&gt;=1.0.0 &lt;3.0.0</code></td>
+        </tr>
+        <tr>
+            <td align="center"><code>'increase'</code></td>
+            <td><code>^2.0.0</code></td>
+        </tr>
+        <tr>
+            <td align="center"><code>'increase-if-necessary'</code></td>
+            <td><code>^2.0.0</code></td>
+        </tr>
+    </tbody>
+</table>
 
 Use the `versioning-strategy` option to change this behavior for supported package managers.
 
 {% data reusables.dependabot.option-affects-security-updates %}
 
-Available update strategies
-
-| Option | Supported by | Action |
-|--------|--------------|--------|
-| `lockfile-only` | `bundler`, `cargo`, `composer`, `mix`, `npm`, `pip` | Only create pull requests to update lockfiles. Ignore any new versions that would require package manifest changes. |
-| `auto` | `bundler`, `cargo`, `composer`, `mix`, `npm`, `pip` | Follow the default strategy described above.|
-| `widen`| `composer`, `npm` | Relax the version requirement to include both the new and old version, when possible. |
-| `increase`| `bundler`, `composer`, `npm`{% ifversion dependabot-increase-version-pip-support %}, `pip`{% endif %}  | Always increase the version requirement to match the new version. |
-| `increase-if-necessary` | `bundler`, `composer`, `npm`{% ifversion dependabot-increase-version-pip-support %}, `pip`{% endif %} | Increase the version requirement only when required by the new version. |
+Available update strategies:
+
+| Ecosystem | Supported _versioning-strategies_ | Default strategy | Additional notes |
+|-----------|-----------------------------------|------------------|------------------|
+| `bundler` | `auto`, `increase`, `increase-if-necessary`, `lockfile-only` | `auto` |
+| `cargo` | `auto`, `lockfile-only` | `auto` |
+| `composer` | `auto`, `increase`, `increase-if-necessary`, `lockfile-only`, `widen` | `auto` |
+| `docker` | N/A | N/A |
+| `github-actions` | N/A | N/A |
+| `gitsubmodule` | N/A | N/A |
+| `gomod` | | N/A | N/A |
+| `gradle` | | N/A | N/A |
+| `maven` | | N/A | N/A |
+| `mix` | `auto`, `lockfile-only` | `auto` |
+| `npm` | `auto`, `increase`, `increase-if-necessary`, `lockfile-only`, `widen` | `auto` |
+| `nuget` | | N/A | N/A |
+| `pip` | `auto`, `increase`, `increase-if-necessary`, `lockfile-only` | `auto` |
+| `pub` | `auto`, `increase`, `increase-if-necessary`, `widen` | `auto` |
+| `terraform` | | N/A | N/A |
+
+**Note:** `N/A` indicates the package manager does not yet support configuring the `versioning-strategy` parameter. The strategy code is open source, so if you'd like a particular ecosystem to support a new strategy, you are always welcome to submit a PR to https://github.com/dependabot/dependabot-core/.
```

Comment:
Typo + following the correct syntax for a note box.

```suggestion
{% note %}

**Note:** `N/A` indicates that the package manager does not yet support configuring the `versioning-strategy` parameter. The strategy code is open source, so if you'd like a particular ecosystem to support a new strategy, you are always welcome to submit a pull request in https://github.com/dependabot/dependabot-core/.

{% endnote %}
```

Reply:
> Typo + following the correct syntax for a note box.